pybv/io.py

```diff
@@ -28,7 +29,9 @@
 
 SUPPORTED_ORIENTS = {'multiplexed'}
 
-SUPPORTED_UNITS = {'V': 1e0, 'mV': 1e3, 'µV': 1e6, 'uV': 1e6, 'nV': 1e9}
+SUPPORTED_VOLTAGE_UNITS = {
```

Comment:
I'm wondering if this should be renamed to `SUPPORTED_VOLTAGE_SCALINGS`?

Reply:
I like that better.